executor/distsql.go

```diff
@@ -437,7 +432,7 @@ func (e *IndexLookUpExecutor) startIndexWorker(ctx context.Context, kvRanges []k
 		finished:     e.finished,
 		resultCh:     e.resultCh,
 		keepOrder:    e.keepOrder,
-		batchSize:    e.maxChunkSize,
+		batchSize:    1, // slow start with small batch size to return first row quickly.
```

Comment:
how about this:

"start from the smallest batch size to reduce the resource consumption and response latency for queries that only need a small amount of the returned result."

@CaitinChen Could you help us to refine this comment?

Reply:
@zz-jason @lysu "Starts slowly from the batch with the minimum size to reduce the resource consumption and response latency of queries that only return a small number of results"?